core-libs/setup/ssr/optimized-engine/request-context.ts

```diff
@@ -9,19 +9,23 @@ import { Request } from 'express';
 import { ExpressServerLogger, ExpressServerLoggerContext } from '../logger';
 import { parseTraceparent } from '../logger/loggers/w3c-trace-context/parse-traceparent';
 
-const getRequestContext = (request: Request) => {
-  return request.res.locals.cx.request;
+const getRequestContext = (request: Request): ExpressServerLoggerContext => {
+  return request.res?.locals.cx.request;
```

Comment:
let's add `?.` on more levels.
```ts
return request.res?.locals?.cx?.request;
```

Why: theoretically it may happen that `cx` object is not defined YET. Then accessing `cx.request` may throw a runtime error

Reply:
Good point, thanks!